config.inc.php

```diff
@@ -119,6 +119,14 @@ function language_hook($PALANG, $language) {
 // Leave blank to send email from the logged-in Admin's Email address.
 $CONF['admin_email'] = '';
 
+// Site admin name
+// This will be used as signature in notification messages
+$CONF['admin_name'] = 'Postmaster';
+
+// Site admin phone number
+// This will be used if a user cannot access his/her email and needs support
+$CONF['admin_phone'] = '';
+
```

Comment:
It seems you don't use $CONF['admin_phone'] anywhere, so please don't add it. We already have enough config options ;-)